data/src/main/java/com/example/data/db/tables/WeatherDataTable.kt

```diff
@@ -1,15 +1,16 @@
-package com.example.data.db.entities
+package com.example.data.db.tables
 
 import androidx.room.ColumnInfo
 import androidx.room.Entity
 
 @Entity(tableName = "weather_data", primaryKeys = ["city_name","date"])
-class WeatherDataEntity(
+class WeatherDataTable(
```

Comment:
Why table instead of entity? 
Now mapper's method looks like this:
fun fromApiToEntity(currentWeatherApi: CurrentWeatherApiModel): WeatherDataTable
But it doesn't convert to table.

Reply:
Fixed. Renamed the classes back to Entity